ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.create-run.tsx

```diff
@@ -55,6 +56,11 @@ import { Separator } from '@/components/ui/separator';
 import { Switch } from '@/components/ui/switch';
 import { useToast } from '@/components/ui/use-toast';
 
+const keyValueSchema = z.object({
+  key: z.string().min(1),
+  value: z.string().min(1),
```

Comment:
So given my above comment, this line should probably drop `.min(1)`. But I'm curious about @mnonnenmacher's opinion here: Do you have a use-case for empty label values?